src/components/GenreSelect.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactElement } from 'react';
+import './GenreSelect.css';
```

Comment:
The usual style import is global in React, you need some styles isolation. (You can refactor it in one the future tasks)

Here is an article about isloation approuches.
https://blog.devgenius.io/best-ways-to-style-a-react-js-application-c818b71f6341

Please use 

I prefer CSS/SCSS modules since it is the most closer to plain css. (webpack and vite support it out of the box https://vitejs.dev/guide/features.html#css-modules).

Reply:
I have used the css modules to add styles isolation